website/docs/docs/cloud/git/managed-repository.md

```diff
@@ -25,3 +25,14 @@ To set up a project with a managed repository:
 6. Click **Create**.
    <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/managed-repo.png" title="Adding a managed repository"/>
 
+## Download managed repository
+
+To download a copy of your managed repository from <Constant name="cloud" /> to your local machine:
+
+1. Navigate to a project using the managed repository feature.
+2. Click **Dashboard** from the main left-side menu.
```

Comment:
So there are two ways to get to a project. I was in the Account settings > Project so I didn't see "Dashboard." Maybe we can orient them in step 1 to selecting "Project" from the left nav as well?